apps/api/v2/src/modules/organizations/conferencing/services/organizations-conferencing.service.ts

```diff
@@ -35,129 +23,9 @@ export class OrganizationsConferencingService {
     private teamsRepository: TeamsRepository,
     private usersRepository: UsersRepository,
     private readonly googleMeetService: GoogleMeetService,
-    private readonly conferencingService: ConferencingService,
-    private readonly platformPlanGuard: PlatformPlanGuard,
-    private readonly isAdminAPIEnabledGuard: IsAdminAPIEnabledGuard,
-    private readonly isOrgGuard: IsOrgGuard,
-    private readonly isTeamInOrg: IsTeamInOrg,
-    private readonly rolesGuard: RolesGuard
+    private readonly conferencingService: ConferencingService
```

Comment:
Question - is per team conferencing installing outside org is possible?
Question 2 - is it possible to install stripe per team aka someone has teams plan and admin wants to install stripe via api for the team.

I am asking this for the future - if we want to allow / someone requests for teams plan an admin to install stripe and conferencing via api for a team, then my guess is that we should rename `OrganizationsConferencingService` to `TeamsConferencingService` and `OrganizationsStripeService` to `TeamsStripeService` so that they can be used for both teams in org and teams outside org.

No need to refactor this now but I just want to know for the future.